builtin/providers/consul/resource_provider.go

```diff
@@ -20,11 +20,34 @@ func Provider() terraform.ResourceProvider {
 			"address": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
+				DefaultFunc: schema.MultiEnvDefaultFunc([]string{
+					"CONSUL_ADDRESS",
+					"CONSUL_HTTP_ADDR",
```

Comment:
We currently check for this (CONSUL_HTTP_ADDR) in the provider_test but not CONSUL_ADDRESS - should we also do that?